src/api/Article.ts

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+export default interface Article {
```

Comment:
così giusto per tenere tutto uguale, secondo me è meglio usare solo named export
```suggestion
export interface Article {
```
e in ```MainApi.ts```
```ts
import { Article } from "./Article"
```

Reply:
Okay